### PR TITLE
This commit will create POST endpoint to insert new comment with erro…

### PR DESCRIPTION
**Pull request by sondos-ahmed**
…r handeling

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -159,6 +159,54 @@ describe("/api/articles/:article_id/comments", () => {
...
+
+  test("POST - 400 responds with an error of bad request", () => {
+    return request(app)
+      .post("/api/articles/banana/comments")
+      .send({})
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Bad request");
+      });
+  });
```
Comment (emilyb93) [inline __tests__/app.test.js:243]: Great testing so far, would just like to see a little bit more testing around the comment object too.
What should happen if we send an object with only a username or only a body and not the other? What happens if we send a comment with an empty string for a body?

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Just a tiny bit more testing but this is looking great so far Sondos!

**Review by Harrison365 — CHANGES_REQUESTED** [review]
Hi Sondos :) 
Plese...
-Test the entire return comment, not just the body.
-You can do this using RETURNING * 

The rest is great :)

**Comment by sondos-ahmed** [comment]
> Hi Sondos :) Plese... -Test the entire return comment, not just the body. -You can do this using RETURNING *
> 
> The rest is great :)
Thanks, I understood to return just the comment body. I am working on it.

File: models/articles.js [hunk 2]
```diff
@@ -41,3 +41,30 @@ exports.selectArticleComments = (article_id) => {
...
+
+exports.insertComment = (article_id, newcomment) => {
+  const { author, body } = newcomment;
+  const values = [article_id, author, body];
+
+  if (typeof body !== "string" || body.length <= 1) {
+    return Promise.reject({ status: 400, message: "Bad request" });
+  }
+
+  if (typeof author !== "string" || author.length <= 1) {
```
Comment (yalicee) [inline models/articles.js:53]: is this a duplicate? If so please remove

**Review by yalicee — APPROVED** [review]
Nice work Sondos :)

File: models/articles.js (hunk 2 above)
Comment (sondos-ahmed) [reply models/articles.js:53]: Thanks Alex, this one check the body in an if statment and the author in another

**Pull request merged**